### PR TITLE
Made bundle compatible with sf3

### PR DESCRIPTION
**Pull request by mvrhov**

### CODE REVIEW

File: sources/lib/DependencyInjection/PommExtension.php [hunk 1]
```diff
@@ -39,9 +39,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services/pomm.yml');
 
-        if ($container->hasParameter('data_collector.time.class')) {
-            $loader->load('services/profiler.yml');
-        }
+        $loader->load('services/profiler.yml');
```
Comment (sanpii) [inline sources/lib/DependencyInjection/PommExtension.php]: This is a probable regression. See #13.

**Comment by mvrhov** [comment]
Then we have a problem.. all *.class parameters have been removed in symfony v3

File: sources/lib/DependencyInjection/PommExtension.php (hunk 1 above)
Comment (mvrhov) [inline sources/lib/DependencyInjection/PommExtension.php]: The problem was that the .class parameter was used. In sf v3, all *.class parameters are gone, and the class name is written directly into the configuration.

Comment (sanpii) [inline sources/lib/DependencyInjection/PommExtension.php]: > P.S. I've run the app in prod mode.. No regression

I confirm, no error. It’s ok for me.

**Comment by sanpii** [comment]
Merged. Thank you.

**Pull request merged**